src/main/java/com/github/onsdigital/babbage/api/filter/DeprecationFilter.java

```diff
@@ -46,7 +46,11 @@ public boolean filter(HttpServletRequest request, HttpServletResponse response)
 
     private void addSunsetHeaders(HttpServletResponse response, DeprecationItem config) {
         response.addHeader(HttpHeaders.DEPRECATION, config.deprecationDate().toString()); 
-        response.addHeader(HttpHeaders.LINK, config.link()); 
+        response.addHeader(HttpHeaders.LINK, formatSunsetLink(config.link())); 
```

Comment:
The formatted link is being generated on each call. Could it be generated once and saved so future calls can simply return the pre-formatted string instead? Not a huge performance hit, but it could add up a bit when babbage is under heavy load.